tutorials/preprocessing/plot_55_setting_eeg_reference.py

```diff
@@ -192,16 +192,6 @@
 # the source modeling is performed. In contrast, applying an average reference
 # by the traditional subtraction method offers no such guarantee.
 #
-# For these reasons, when performing inverse imaging, *MNE-Python will
-# automatically average-reference the EEG channels if they are present and no
-# reference strategy has been specified*. If you want to perform inverse
-# imaging and do not want to use an average reference (and hence you accept the
-# risks presented in the previous paragraphs), you can force MNE-Python to
-# relax its average reference requirement by passing an empty list to
-# :meth:`~mne.io.Raw.set_eeg_reference` (i.e., by calling
-# ``raw.set_eeg_reference(ref_channels=[])``) prior to performing inverse
-# imaging.
```

Comment:
you remove this as it's not true anymore? or MNE will already give you a meaningful message?

Reply:
yes, it's not true anymore - and MNE puts a ValueError when users try to do inverse without an average ref, see https://github.com/mne-tools/mne-python/pull/7474/commits/b00a3f806a8ed2ee3659238e6f36f72b153105dc#r394477027

Reply:
I'd be inclined to still say *something* here. Maybe this?

```suggestion
# For these reasons, when performing inverse imaging, *MNE-Python will
# raise a ``ValueError`` if there are EEG channels present and
# something other than an average
# reference strategy has been specified*.
```

Although it seems odd to me that it's no longer possible to override using `[]`

Reply:
Change looks reasonable to me

I don't know if it was ever actually possible to relax the average ref requirement, that sounds like a documentation error to me

Reply:
(by "change" looking reasonable I mean these lines with your suggestion incorporated @drammock -- the documentation error looks like it will be removed even with your suggested changes  applied)